features/src/main/feature/camel-features.xml

```diff
@@ -1542,6 +1542,11 @@
         <bundle dependency='true'>wrap:mvn:com.pubnub/pubnub-gson/${pubnub-version}</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-pubnub/${project.version}</bundle>
     </feature>
+    <feature name='camel-pulsar' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <bundle dependency='true'>wrap:mvn:org.apache.pulsar/pulsar-client-admin/${pulsar-version}$overwrite=merge&amp;Export-Package=org.apache.pulsar*;version=${pulsar-version};resolution:=optional</bundle>
```

Comment:
Warning, I see 2 problems with your wrap URL:

1. The jar is not an OSGI bundle so `overwrite=merge` is not needed, as it generates for scratch
2. The resolution only makes sense for import packages